jr-annotation-support/src/test/java/com/fasterxml/jackson/jr/annotationsupport/ASTestBase.java

```diff
@@ -11,8 +13,50 @@
 {
     protected enum ABC { A, B, C; }
 
+    /**
+     * enum that uses {@link JsonProperty} to alias values
+     */
     protected enum ABCRename { @JsonProperty("A1") A, @JsonProperty("B1") B, C; }
 
+    /**
+     * enum that uses a {@link JsonValue} method to alias its values and a {@link JsonCreator} to
+     * deserialize
+     */
+    protected enum ABCJsonValueJsonCreator
```

Comment:
Probably does not need to be in test base class but actual test class? (unless it is shared across tests)

Reply:
With that in mind, I've moved `ABCRename` too and deleted the unused `ABC`